python/lib/sift_py/ingestion/rule/config_test.py

```diff
@@ -158,9 +163,9 @@ def test_rule_namespace():
     assert valid_namespace_rule.name == "valid_rule"
     assert valid_namespace_rule.description == "A rule in a namespace"
     assert valid_namespace_rule.expression == "$1 > 10"
-    assert valid_namespace_rule.action.assignee == "bob@example.com"
-    assert valid_namespace_rule.action.tags == ["foo", "bar"]
-    assert valid_namespace_rule.action.kind() == RuleActionKind.ANNOTATION
+    assert valid_namespace_rule.action.assignee == "bob@example.com"  # type: ignore
+    assert valid_namespace_rule.action.tags == ["foo", "bar"]  # type: ignore
+    assert valid_namespace_rule.action.kind() == RuleActionKind.ANNOTATION  # type: ignore
```

Comment:
What errors are we getting without the `type: ignore`?

Reply:
Pyright is upset about the optional type:
```
  /Users/ailinyu/repos/sift/python/lib/sift_py/ingestion/rule/config_test.py:166:40 - error: Cannot access attribute "assignee" for class "RuleAction"
    Attribute "assignee" is unknown (reportAttributeAccessIssue)
  /Users/ailinyu/repos/sift/python/lib/sift_py/ingestion/rule/config_test.py:166:40 - error: "assignee" is not a known attribute of "None" (reportOptionalMemberAccess)
  /Users/ailinyu/repos/sift/python/lib/sift_py/ingestion/rule/config_test.py:167:40 - error: Cannot access attribute "tags" for class "RuleAction"
    Attribute "tags" is unknown (reportAttributeAccessIssue)
  /Users/ailinyu/repos/sift/python/lib/sift_py/ingestion/rule/config_test.py:167:40 - error: "tags" is not a known attribute of "None" (reportOptionalMemberAccess)
  /Users/ailinyu/repos/sift/python/lib/sift_py/ingestion/rule/config_test.py:168:40 - error: "kind" is not a known attribute of "None" (reportOptionalMemberAccess)
```
I thought in this case since this is a test and these *should* be populated if the test is working, it would be better to ignore and let this just error out if something were to break in the test down the line. Open to other thoughts though, lmk what you think!

Reply:
(Or I guess it's actually that those are fields of child classes, not optional)

Reply:
[Can we just have pyright ignore this rule completely](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#sample-pyprojecttoml-file)?